CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 5.4.1
+- Correctly perform healthchecks against ES 5.x nodes
```

Comment:
My reading of the code makes me believe this changelog entry to be inaccurate.

It changes how we sniff for http nodes, doesn't it, not how we do health checks?